### ISSUE TITLE
Doesn't show case battery level.

### ISSUE THREAD

**Issue by Zoikkimc** [steam3d/MagicPods-Windows#170]

**Comment by Zoikkimc** [comment]
![image](https://user-images.githubusercontent.com/97616927/149172801-ca04cba3-6bdd-4e81-afda-df38c5ae514d.png)

**Comment by steam3d** [comment]
Hi. Everything fine. iPhone shows battery case about 30-60 sec then it hides.

**Comment by steam3d** [comment]
Hi. Everything fine. iPhone shows case battery about 30-60 sec then it hides.

**Comment by steam3d** [comment]
No.  Because the AirPods's case does not have Bluetooth and can't send battery information without AirPods inside.
